lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -132,9 +130,11 @@ class EffectiveClangContext {
   /// Retrieve the unresolved context name.
   StringRef getUnresolvedName() const {
     assert(getKind() == UnresolvedContext);
-    return StringRef(Unresolved.Data, Unresolved.Length);
+    return StringRef(Unresolved.Data, UnresolvedLength);
   }
 };
+static_assert(sizeof(EffectiveClangContext) <= 2 * sizeof(void *),
```

Comment:
Do we support a 32-bit *host*? There's no CI for 32-bit Linux AFAIK (but would be nice to have assuming it works)